src/Pate/Equivalence.hs

```diff
@@ -93,17 +92,9 @@ instance Semigroup EquivalenceStatus where
 instance Monoid EquivalenceStatus where
   mempty = Equivalent
 
-type StatePredSpec sym arch = SimSpec sym arch (PES.StatePred sym arch)
+type DomainSpec sym arch = SimSpec sym arch (PED.EquivalenceDomain sym arch)
```

Comment:
The 'SimSpec' pattern is somewhat general and exposed here, since the inner domains need to be instantiated via 'bindSpec' before they can actually be used. The only thing I can think of that would potentially justify defining this as a 'newtype' would be to formally distinguish when a 'DomainSpec' is being interpreted as a pre-domain vs. post-domain (i.e. define PreDomain and PostDomain types which are both a 'DomainSpec' under the hood).